rotors_gazebo/resource/lee_controller_neo9.yaml

```diff
@@ -0,0 +1,5 @@
+# Lee position controller parameters
+position_gain: {x: 8, y: 8, z: 17}
+velocity_gain: {x: 6, y: 6, z: 10}
+attitude_gain: {x: 4, y: 4, z: 2}
```

Comment:
I know the other files don't all follow this convention yet, but make the values 4.0, 17.0, ... in the lines above